packages/hop-node/src/watchers/CommitTransferWatcher.ts

```diff
@@ -107,6 +111,8 @@ class CommitTransfersWatcher extends BaseWatcherWithEventHandlers {
     if (this.isL1) {
       return
     }
+
+    const commitTransferWatcherPollInterval = 6 * 10 * 1000
```

Comment:
Instead of creating a new variable, you can assign `this.pollIntervalSec = 60` in constructor